packages/web/src/store/application/ui/buy-audio/sagas.ts

```diff
@@ -77,9 +79,12 @@ const SOLANA_CLUSTER = process.env.REACT_APP_SOLANA_WEB3_CLUSTER
 const ERROR_CODE_INSUFFICIENT_FUNDS = 1 // Error code for when the swap fails due to insufficient funds in the wallet
 const ERROR_CODE_SLIPPAGE = 6000 // Error code for when the swap fails due to specified slippage being exceeded
 const SLIPPAGE = 3 // The slippage amount to allow for exchanges
-const MIN_PADDING = 0.00005 * LAMPORTS_PER_SOL // Buffer for SOL in wallet
 let _jup: Jupiter
 
+const MEMO_MESSAGES = {
+  [OnRampProvider.COINBASE]: 'In-App Coinbase $AUDIO Purchase'
```

Comment:
@piazzatron does this memo look good?

Reply:
Might be nice to have a schema to it, like `ONRAMP:PURCHASE:COINBASE`?